CONTRIBUTING.md

```diff
@@ -1,75 +1,46 @@
-# Contributing to the Write-A-Thon Writing Challenge
+# 🤝 Contributing to the WriteTech Hub Write-A-Thon Challenge Series
```

Comment:
I'm thinking we should remove the emoji's for accessibility sakes. Find out about this so we can proceed with best practice

Reply:
Sure thing